_includes/head.html

```diff
@@ -1,11 +1,13 @@
 {% assign chapter = page.url | regex_extract: '^/([^/]+)/' %}
+{% assign description = page.excerpt | strip_html | strip_newlines | truncate: 160 %}
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
-  <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
+  <title>{% if chapter == 'apps' %}{{ site.api_title }}{% else %}{{ site.blog_title }}{% endif %}</title>
+  <meta name="description" content="{% if page.excerpt %}{{ description }}{% elsif chapter == 'apps' %}{{ site.api_description }}{% else %}{{ site.blog_description }}{% endif %}">
+  <meta name="keywords" content="{% if chapter == 'apps' %}{{ site.api_keywords }}{% else %}{{ site.blog_keywords }}{% endif %}">
 
```

Comment:
This solution is far from optimal:

1. If one opens up the "themes" chapter, he also gets the SEO texts from the blog.
2. This will get more and more clumsy if one needs more different SEO texts for different chapters.

A better solution would be, to have a file `_data/seo.yml` somewhat like this:

```yml
apps:
  title: This is the API
  description: >
    Foobar
  keywords: ["a", "b", "c"]
blog:
  title: This is the blog
  description: >
    Foobar
  keywords: ["a", "b", "c"]
themes:
  title: This is the themes documentation
  description: >
    Foobar
  keywords: ["a", "b", "c"]
#...
```

Now you can alter this part (not tested, but should work somewhat like this):

```html
<title>{% if site.data[chapter].title %}{{ site.data[chapter].title }}{% else %}ePages{% endif %}</title>
```

The same then for `<meta name="description"/>` and `<meta name="keywords"/>`